CONTRIBUTING.md

```diff
@@ -18,4 +18,4 @@ Bugs or remarks are tracked as GitHub issues. You can create an issue and choose
 
 ### Adding a new page or changing an existing page
 
-See our [How to contribute page](how_to_contribute).
+See our [How to contribute page](https://rdmkit.elixir-europe.org/how_to_contribute).
```

Comment:
I think we only use the page name and not the full url, @bedroesb correct?

Reply:
In theory yes, but I think many people end up on this page in its 'markdown' form on GitHub, and than people might want to be directed to the actual RDMkit page with contribute info. But if they find the page thought RDMkit, than the old linking is better

Reply:
I used the full URL because if you visit https://github.com/elixir-europe/rdmkit/blob/master/CONTRIBUTING.md from GitHub the relative link doesn't work.

Is `CONTRIBUTING.md` also served by the RDM-Kit website, or only for viewing in GitHub? If it's the former, I'm not sure what the best solution is here.

Reply:
oh, yes. sorry. my mistake.